packages/eslint-config/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
```

Comment:
Why?

Reply:
trim_trailing_whitespace = false in `*.md` files?
copy from other repo which I  forgot...

Reply:
2333